docs/pages/enroll-resources/application-access/okta/app-and-group-sync.mdx

```diff
@@ -0,0 +1,111 @@
+(optional), *Level 3. User sync* and *Level 4. App and Group sync* enrollment in the [Okta
+integration setup guide](./integration-enrollment.mdx).
+
+## Synchronizing Access Lists from Okta groups and applications
+
+The Okta Access List synchronizer will look for any Okta group with members or any Okta application
+with individual assignments that matches configurable filters that you can optionally supply during
+Okta integration enrollment. Access Lists will not be created for groups or applications without
+any assignments.
+
+The synchronizer will create the following resources for each matched Okta group or application:
+
+- A role for members that grants access to the group/application.
+- A role for owners that grants the ability to review access to the group/application.
+- An Access List representing membership to the group/application.
+- Members for the Access List.
+
+It should be noted that the Access List sync waits until the Okta groups and Okta applications
+has finished syncing as Teleport resources, so it may not start synchronizing immediately on startup.
+
+## Handling nested Access Lists
+
+### Synchronization from Teleport to Okta
+
+Teleport supports nested Access Lists, where an Access List can include other Access Lists as members,
+creating a hierarchical structure. However, Okta does not support nested groups.
+To accommodate this limitation, the synchronizer flattens nested Access Lists when synchronizing from Teleport to Okta.
+
+Members in Access Lists, from all levels of nesting, are aggregated into a single, flat list of members when
+synchronized to Okta. This ensures that all users who should have access according to the Teleport hierarchy
+are included in the corresponding Okta group or application.
+
+**Example**:
+  - **Teleport Structure**:
+    - **Access List A**:
+      - Members: User1
+      - Nested Members: Access List B
+    - **Access List B**:
+      - Members: User2, User3
+  - **Okta Representation**:
+    - Group/Application for Access List A:
+      - Members: User1, User2, User3
+
+By flattening the Teleport hierarchy, all users receive the permissions associated with the root-level Access List in Okta.
+
+### Synchronization from Okta to Teleport
+
+When synchronizing from Okta to Teleport, the synchronizer handles the flattened structure from Okta
+by attempting to map it back to the Teleport hierarchy:
+
+- **Comparing with Teleport Hierarchy**: The flattened list of members from Okta is compared against the existing
+Access List hierarchy in Teleport.
+- **Adding New Members**:
+  - **New Users**: If there are new users added in Okta that are not present in the Teleport Access List hierarchy,
+    these users are added as members at the root-level Access List in Teleport.
+  - **Maintaining Hierarchy**: This approach maintains the hierarchical structure within Teleport while ensuring
+    that access changes made in Okta are reflected appropriately.
```

Comment:
I think the dense outline structure here is a little distracting I would use paragraphs instead.